dvol_test.go

```diff
@@ -1 +0,0 @@
-package main
```

Comment:
Is this now an empty file, or is it deleted? If not deleted, could it be deleted?